build/azure-pipelines/distro-build.yml

```diff
@@ -13,22 +13,22 @@ steps:
 - task: AzureKeyVault@1
   displayName: 'Azure Key Vault: Get Secrets'
   inputs:
-    azureSubscription: 'vscode-builds-subscription'
-    KeyVaultName: vscode
+    azureSubscription: 'azuredatastudio-adointegration'
+    KeyVaultName: ado-secrets
 
 - script: |
     set -e
 
     cat << EOF > ~/.netrc
     machine github.com
-    login vscode
+    login azuredatastudio
     password $(github-distro-mixin-password)
     EOF
 
-    git config user.email "vscode@microsoft.com"
-    git config user.name "VSCode"
+    git config user.email "andresse@microsoft.com"
```

Comment:
We really should set up our own e-mail for these kind of things. Using specific alias' just makes it harder to discover issues if you're not available

Reply:
This specific email doesn't really matter, its just used as the git identity, there would be no reason to use this email to contact anything.